tests/ci/merge_pr.py

```diff
@@ -101,10 +103,24 @@ def is_approved(self, team: List[NamedUser]) -> bool:
             last_changed = datetime.strptime(
                 commit.stats.last_modified, "%a, %d %b %Y %H:%M:%S GMT"
```

Comment:
maybe timezone is not same for last_changed & approved_at? or is it GMT for both?

Reply:
submitted_at is always in UTC (at least, it looks like it, otherwise we are doomed) https://docs.github.com/en/rest/pulls/reviews?apiVersion=2022-11-28#list-reviews-for-a-pull-request

And I know that GMT != UTC, but it has the same timeshift at least ¯\\\_(ツ)_/¯